cunumeric/config.py

```diff
@@ -78,6 +78,7 @@ def destroy(self):
 # Match these to CuNumericOpCode in cunumeric_c.h
 @unique
 class CuNumericOpCode(IntEnum):
+    ADVANCED_INDX = _cunumeric.CUNUMERIC_ADVANCED_INDEXING
```

Comment:
why not use the same string as C++, `ADVANCED_INDEXING`?

Reply:
Sure, I will do that